### PR TITLE
MG-93 - Add bootstraps charts support

### PR DESCRIPTION
**Pull request by Musilah**
# What type of PR is this?

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Dependency Update

## What does this do?
This adds more charts from bootstrap

## Which issue(s) does this PR fix/relate to?

- Related Issue #93 

## Have you included tests for your changes?

- [ ] Yes
- [X] No 

## Did you document any new/modified functionality?

- [ ] Yes
- [X] No 

### Notes
The charts being added include: 
- value card
- value and chart card
- alarm count
- entity count
- alarm table
- entity table
- label card
- progress bar

### CODE REVIEW

**Comment by ianmuchyri** [comment]
Fix the height styling. There is too much unnecessary space at the bottom

image

File: ui/web/static/js/singleDashboard.js [hunk 1]
```diff
@@ -223,10 +226,37 @@ const resizeObserver = new ResizeObserver((entries) => {
...
-      width: itemContentWidth,
-      height: itemContentHeight,
+
+    var chart = echarts.getInstanceByDom(contentEl);
+    if (chart) {
+      chart.resize({
+        width: itemContentWidth,
+        height: itemContentHeight,
       });
+    } else {
```
Comment (ianmuchyri) [inline ui/web/static/js/singleDashboard.js:236]: This only works for widgets with cards in them. For the rest with the tables and progress bar it does not work well check on that

image

File: ui/web/templates/dashboard.html [hunk 2]
```diff
@@ -124,4 +132,4 @@ <h5 id="widgetsCanvasLabel">Widgets</h5>
       <script src="/js/singleDashboard.js"></script>
     </body>
   </html>
-{{ end }}
+{{ end }}
```
Comment (ianmuchyri) [inline ui/web/templates/dashboard.html:135]: add empty line

**Review by ianmuchyri — CHANGES_REQUESTED** [review]

**Review by 1998-felix — APPROVED** [review]
The resizing of the widgets is still not optimal but is being handled by issue https://github.com/absmach/magistrala-ui/issues/129

**Review by ianmuchyri — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**